client/src/components/ProductListing/ProductListing.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+export default function ProductListing({
+  img,
+  brand,
+  model,
+  price,
+  sale = false,
+}) {
+  return (
+    <>
+      <ProductListElement>
+        <ProductImage src={img} />
+        <ProductInfo>
+          <ProductBrand>{brand}</ProductBrand>
+          <ProductModel>{model}</ProductModel>
+        </ProductInfo>
+        <ProductMeta>
+          <ProductPrice sale={sale}>
+            ab <span>{price} €</span>
+          </ProductPrice>
+        </ProductMeta>
+      </ProductListElement>
+    </>
+  )
+}
+
+ProductListing.propTypes = {
+  img: PropTypes.string,
+  brand: PropTypes.string,
+  model: PropTypes.string,
+  price: PropTypes.string,
+  sale: PropTypes.bool,
+}
+
+const ProductListElement = styled.li`
+  font-family: Source Sans Pro, sans-serif;
+  color: #6f7172;
```

Comment:
Please consider css variables

Reply:
I've added globalstyles.js and css variables